mod/groups/lib/discussion.php

```diff
@@ -19,6 +19,7 @@ function discussion_handle_all_page() {
 		'full_view' => false,
 		'no_results' => elgg_echo('discussion:none'),
 		'preload_owners' => true,
+		'preload_containers' => true,
```

Comment:
Why load containers here? Aren't just the owners displayed in the list, and not the groups?